src/components/Select/Select.stories.js

```diff
@@ -9,7 +9,9 @@ const SelectTemplate = (args) => ({
     SbSelect,
   },
 
-  props: Object.keys(args),
+  setup() {
+    return { ...args }
+  },
```

Comment:
This is to solve this problem in the storybook

https://user-images.githubusercontent.com/7618411/226989475-71aace3c-c0be-484c-8e2f-b7abf418672f.mov